middle_end/flambda/simplify/simplify_set_of_closures.ml

```diff
@@ -251,6 +251,9 @@ end = struct
       denv
       |> DE.enter_set_of_closures
       |> DE.increment_continuation_scope_level_twice
+      (* Even if we are not rebuilding terms we should always rebuild it
```

Comment:
"rebuild it" -> "rebuild them"

Reply:
(also, in fact, extend this comment slightly to say why we need to rebuild them)

Reply:
Done